### ISSUE TITLE
Improvements to messaging and tone of voice

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#1295]
List of improvements to be confirmed.

**Comment by asimonw** [comment]
@fannyBkane @Kimberley-Heath would you like to list the changes in this issue or is it more practical to link to the mockup? I'm only talking about changes to existing screens. New screens will automatically use the copy from the confirmed prototype.

**Comment by fannyBkane** [comment]
@asimonw listing all the changes here sounds good, and we can link to particular states from here too.

**Comment by fannyBkane** [comment]
@asimonw In order to also include microcopy changes like the one as follows:

>'Settings' become 'My personal settings' in the burger menu on mobile and the drop down on desktop, and the header of the settings page is changed to 'My personal settings' instead of edit user details

Could we edit the title of the issue to a more general 'copy and microcopy changes'?

**Comment by fannyBkane** [comment]
### **Primary/secondary users screens**

**Login page**
'Collaborate more, solve problems faster' changes to ... tbc :)

**Create password page**
`<input>` 'Enter password' changes to 'Choose password'

**Explore>select tag modal**
'Search all tags' changes to 'Clear filters'

**User profile**
`<H2>` 'Challenges' changes to 'My challenges'

**Nav**
'Settings' link changes to 'My personal settings'

**(user)edit settings page**
`<H1>` 'Edit settings' changes to 'My personal settings'

### **Admin screens**
**(user)Settings page**
`<H1>` 'Edit user details' changes to 'Edit user settings' 

**(org)edit profile page**
`<H1>` 'Edit org profile' changes to 'Edit organisation settings'

**Comment by sophielevens** [comment]
Hi @fannyBkane, could you let us know what copy change you would like on the login page?

> Login page
'Collaborate more, solve problems faster' changes to ... tbc :)

Thanks 👍

**Comment by fannyBkane** [comment]
Hey @sophielevens actually this is unlikely to be solved in this sprint. We should remove it, would you agree @asimonw ?

**Comment by asimonw** [comment]
@sophielevens Yes, you can leave that one for now, we will create a new issue for it. We're still crafting that one ;)

**Comment by fannyBkane** [comment]
Nice job! One small thing I've spotted:

- On the **User profile** Interests should be ‘My interests’

And also, realised... Can this be also applied to the login page?

> Create password page
> <input> 'Enter password' changes to 'Choose password'
> 

Regarding the items below, I see there hasn't been changes made, but actually let's not worry about it as we're going to review the select tags modals in the next sprint!

> Primary/secondary users screens
> Challenge > Create
> H1 changes to 'Choose tags'
> Button text changes to 'Save'
> 
> Challenge > Edit
> H1 changes to 'Edit tags'
> Button text changes to 'Save'

Thank you!

**Comment by asimonw** [comment]
@sophielevens Seems to be all fine for now, except for the small issue Fanny spotted (see previous comment).
